pgcli/packages/sqlcompletion.py

```diff
@@ -27,6 +28,15 @@ def suggest_type(full_text, text_before_cursor):
 
     identifier = None
 
+    def strip_named_query(txt):
+        pattern = re.compile(r'\\(n|nd|ns\s+[a-z]+)\s+')
```

Comment:
Yes but that's only for checking the match, when you call `re.sub()` it still uses the original regex which can replace `\n` anywhere in the query.